pkg/k8sclient/client.go

```diff
@@ -105,11 +116,23 @@ func NewForRestConfig(c *rest.Config, schemes ...func(scheme *runtime.Scheme) er
 	return &kc, nil
 }
 
-type Option func(*rest.Config)
+type Option func(*K8sClient)
 
 func WithTimeout(timeout time.Duration) Option {
-	return func(rc *rest.Config) {
-		rc.Timeout = timeout
+	return func(k *K8sClient) {
+		k.timeout = &timeout
+	}
+}
+
+func WithSchemes(schemes ...func(scheme *runtime.Scheme) error) Option {
+	return func(k *K8sClient) {
+		k.schemes = schemes
+	}
+}
+
+func WithPriorityUseInCluster() Option {
+	return func(k *K8sClient) {
+		k.priorityUseInCluster = true
```

Comment:
wrong translation.

`WithPreferredToUseInClusterConfig`

Reply:
adjusted, and add comment for this option